jenkins/build-and-package.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+#
+# Jenkins build-and-package
+#
+
+set -eo pipefail
+source ~/.bash_profile
+
+# Vars
+local_indicator=$1
+
+#
+# Build
+#
+
+cd "${WORKSPACE}/${local_indicator}" || exit
+
+# Set up venv
+python -m venv env
+source env/bin/activate
+pip install ../_delphi_utils_python/.
+pip install .
+
+#
+# Package
+#
+
+cd "${WORKSPACE}" || exit
+
+# Create .tar.gz for deployment
+# TODO: Switcheroo to `artifacts` after this goes live!
```

Comment:
same same

Reply:
Do it all in this PR. I'll take a backup of the artifacts dir on the Jenkins server as well, just in case.